libraries/Microsoft.Bot.Builder.AI.Orchestrator/OrchestratorRecognizer.cs

```diff
@@ -0,0 +1,346 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using AdaptiveExpressions.Properties;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Builder.Dialogs.Adaptive.Recognizers;
+using Microsoft.Bot.Builder.TraceExtensions;
+using Microsoft.Bot.Schema;
+using Microsoft.Orchestrator;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder.AI.Orchestrator
+{
+    /// <summary>
+    /// Class that represents an adaptive Orchestrator recognizer.
+    /// </summary>
+    public class OrchestratorRecognizer : Recognizer
+    {
+        /// <summary>
+        /// The Kind name for this recognizer.
+        /// </summary>
+        [JsonProperty("$kind")]
+        public const string Kind = "Microsoft.OrchestratorRecognizer";
```

Comment:
is this not available through reflection - it looks like the class name

Reply:
No change. Just keeping this similar to how the rest of the adaptive is done :)